Haneke/UIButton+Haneke.swift

```diff
@@ -37,16 +37,13 @@ public extension UIButton {
     }
     
     public func hnk_setImageFromFetcher(fetcher : Fetcher<UIImage>, state : UIControlState = .Normal, placeholder : UIImage? = nil, format : Format<UIImage>? = nil, failure fail : ((NSError?) -> ())? = nil, success succeed : ((UIImage) -> ())? = nil){
+        self.setImage(placeholder, forState: state)
```

Comment:
@hpique @joanromano nilling out the image in prepareForReuse has a considerate impact on the performance as it seems like it doesn't use the cached image.  I've inspected with Time profiler and when nilling the image double/triple the executiong time on setImageFromURL